src/android/Capture.java

```diff
@@ -59,7 +59,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 import android.net.Uri;
 import android.os.Environment;
 import android.provider.MediaStore;
-import androidx.core.content.FileProvider;
```

Comment:
Our next major version of cordova-android will force using AndroidX, so it's probably best if we stick with androidx here.

Cordova-android@9 supports `AndroidX` via the `AndroidXEnabled` preference.

Reply:
I tried using androidx, but apparently it is rather troublesome to enable androidx in the media-capture plugin. I think a better solution would be using the old framework like other plugins, and let a cordova plugin called [cordova-plugin-androidx-adapter](https://github.com/dpa99c/cordova-plugin-androidx-adapter) take care of migrating to AndroidX.

Reply:
Hmm, what kind of trouble?

Unfortunately I don't think requiring users to install `cordova-plugin-androidx-adapter` because of an Apache maintained plugin is acceptable, even though they will very likely require it for other third-party plugins.

I can do my best to assist, but I only have beginner to moderate knowledge of Android native development. You can also try reaching out to our [Slack Community](http://slack.cordova.io/) and our [Dev Mailing List](https://cordova.apache.org/contact/) for assistance.

Note that the Dev Mailing list is for discussions about developing Apache maintained cordova packages only, so as long as the subject is about this plugin for example, it can be used to try to gain some guidance about how to solve a particular problem.

Reply:
I managed to update to using androidx.